futures-util/src/try_stream/try_skip_while.rs

```diff
@@ -0,0 +1,117 @@
+use core::marker::Unpin;
+use core::mem::PinMut;
+use futures_core::future::TryFuture;
+use futures_core::stream::{Stream, TryStream};
+use futures_core::task::{self, Poll};
+use pin_utils::{unsafe_pinned, unsafe_unpinned};
+
+/// A stream combinator which skips elements of a stream while a predicate
+/// holds.
+///
+/// This structure is produced by the `Stream::skip_while` method.
```

Comment:
This should be a link

Reply:
Done, but this isn't a link in any of the other combinators I could find.